.github/workflows/acceptance.yml

```diff
@@ -124,7 +124,7 @@ jobs:
         run: make run-acceptance
 
       - name: Inspect failure
-        if: failure()
+        if: failure() && command -v devservices
```

Comment:
heh this isn't going to work -- I meant you'd have to do this in the shell

Reply:
dang I just pushed a commit to test, rip